src/core/components/accordion/styles.ts

```diff
@@ -1,11 +1,83 @@
 import { css } from "@emotion/core"
 import { border } from "@guardian/src-foundations/palette"
-import { headline } from "@guardian/src-foundations/typography"
+import { headline, textSans } from "@guardian/src-foundations/typography"
+import { space } from "@guardian/src-foundations"
+import { background, neutral } from "@guardian/src-foundations/palette"
 
 export const accordion = css`
-	border: 1px solid ${border.primary};
+	border-top: 1px solid ${border.primary};
+	border-bottom: 1px solid ${border.primary};
+	padding: ${space[2]}px 0 ${space[4]}px ${space[2]}px;
+
+	button {
+		background: none;
+		color: inherit;
+		border: none;
+		padding: 0;
+		font: inherit;
+		cursor: pointer;
+		outline: inherit;
+	}
```

Comment:
Shall we extract this into a separate `const`?

Reply:
Sure thing - I have create a 'normaliser' style for this.